core/kernel/shared/require.rb

```diff
@@ -481,6 +481,21 @@
     end
```

Comment:
The version guard above needs to be adapted to be `ruby_version_is "2.3"..."2.5" do`